libsolidity/formal/SMTLib2Interface.cpp

```diff
@@ -156,6 +148,15 @@ string SMTLib2Interface::toSmtLibSort(Sort const& _sort)
 	}
 }
 
+string SMTLib2Interface::toSmtLibSort(vector<SortPointer> const& _sorts)
+{
+	string ssort("(");
+	for (auto const& sort: _sorts)
+		ssort += toSmtLibSort(*sort) + ' ';
+	ssort += ')';
```

Comment:
For consistency perhaps use double quotes in both places. If we are concerned by the speed, then we should replace it everywhere to use single quotes for single characters.